src/components/AreasContainer/AreasContainer.test.js

```diff
@@ -1,20 +1,101 @@
 import React from 'react';
-import { render, fireEvent } from '@testing-library/react';
-import '@testing-library/jest-dom/extend-expect'
+import { render, waitForElement, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import '@testing-library/jest-dom/';
 import { BrowserRouter as Router, Link } from "react-router-dom";
 import AreasContainer from './AreasContainer';
 
-describe("AreasContainer", () => {
-  it("should display the welcome message", () => {
+describe("AreasContainer - Navigation Display", () => {
```

Comment:
This should just be tested in the Nav component where this data is actually being handled.